x-pack/test_utils/jest/config.integration.js

```diff
@@ -4,16 +4,20 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { RESERVED_DIR_JEST_INTEGRATION_TESTS } from '../../../src/dev/constants';
 import config from './config';
 
 export default {
   ...config,
   testMatch: [
-    '**/integration_tests/**/*.test.js',
-    '**/integration_tests/**/*.test.ts',
+    `**/${RESERVED_DIR_JEST_INTEGRATION_TESTS}/**/*.test.js`,
+    `**/${RESERVED_DIR_JEST_INTEGRATION_TESTS}/**/*.test.ts`,
```

Comment:
These two lines can be combined using `*.test.{js,ts}`

Reply:
Thanks for the tip! Done.